crates/sui-json-rpc/src/read_api.rs

```diff
@@ -121,6 +123,52 @@ impl SuiRpcModule for ReadApi {
 
 #[async_trait]
 impl RpcFullNodeReadApiServer for FullNodeApi {
+    async fn get_move_function_args_tagging(
+        &self,
+        package: ObjectID,
+        module: String,
+        function: String,
+    ) -> RpcResult<Vec<String>> {
```

Comment:
Wondering why we use strings here to represent the tags instead of an enum?

Reply:
I was thinking this function will only be called from typescript side so the enum type will not help much, but I will do that.